file_handling/load.py

```diff
@@ -1,16 +1,16 @@
 from mantid.simpleapi import AlignAndFocusPowderFromFiles, NormaliseByCurrent, SetSample, ConvertUnits
 
 
-def load(ws_name, input_files, geometry=None, material=None, mass_density=None, **align_and_focus_args):
+def load(ws_name, input_files, Geometry=None, ChemicalFormula=None, MassDensity=None, **align_and_focus_args):
```

Comment:
python standard is to use snake case for variable / argument naming. Is there a reason for using Camel case here?

Reply:
The pseudo-Camel case is to follow with Mantid names for these. My thought was to keep that consistent, possibly so if you make a dictionary you can pass it to this or Mantid algorithms alike.

I can change it back if you disagree, no problem. Thoughts?

Reply:
Because this is destined to be a mantid algorithm then it would be the algorithm properties that get capitalised for example in AlignAndFocusPowderFromFiles the properties that are exposed to the users are defined with [capital letters](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py#L79). However, when the user input is extracted from these at run time, they are stored in [snake_case variables](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py#L278)

So I think in this case (unless I've misunderstood) then these should be snake case as the algorithm properties acts as an indirect map between them

Reply:
:+1: I'll change to snake case, thanks!